activity/writer.go

```diff
@@ -0,0 +1,17 @@
+package activity
+
+import (
+	"context"
+	"strings"
+)
+
+// Writer logs the output as a call to Record per Write.
+type Writer struct {
```

Comment:
This could wrap around an existing writer (decorate it w/ the ability to record to the activity journal).

Reply:
That seems like an additional feature we don't need yet, not a change to the current solution.

Reply:
I may not be clear on what you're suggesting. What does "decorate it w/ the ability to record to the activity journal" mean? That it would record to the activity journal while also recording to its original destination?

That can also be accomplished with this writer plus https://golang.org/pkg/io/#MultiWriter.

Reply:
This is specifically a Writer interface to the activity journal.

Reply:
Ah, didn't know about MultiWriter. Nevermind then.